### DISCUSSION TITLE
[Feature Request] /afk command

### DISCUSSION THREAD

**Discussion by DerpDerpling** [PurpurMC/Purpur#828] [__Dismissed__] [answered]
adds a /afk command that integrates with purpur's afk system to set themselves as afk

**Comment by BillyGalbreath** [comment, marked as answer]
The afk system is built on top of Mojang's idle timeout. Performing a command (such as `/afk`) would reset the idle time, resulting in the player not being afk.

To make things work how you want the entire system would need to be rewritten, at which point it would be easier to just use one of the many plugins that offer a better afk system that already do these things.

**Reply by DerpDerpling** [reply]
ah ok, didnt know it would be that hard for that. ill just use an afk plugin
